tests/conftest.py

```diff
@@ -88,3 +103,50 @@ def fuzzy_checker(output_directory: Path) -> Generator[FuzzyChecker, None, None]
     yield checker
 
     checker.save_diagnostic_output(output_directory)
+
+
+@pytest.fixture(scope="session")
+def config() -> dict[str, Any]:
```

Comment:
This looks like it just got copy/pasted from somehwere?

Reply:
This config used to be in the integration directory but now it's shared across /integration and /release so I put it in here.